components/api/core/api_impl.bal

```diff
@@ -499,30 +499,54 @@ returns http:Response {
 # + orgName - Organization name
 # + imageName - Image ID
 # + return - http response (200 if success, 405 or 500 otherwise)
-public function updateImage (http:Request updateImageReq, string orgName, string imageName, gen:ImageUpdateRequest updateImageBody) returns http:Response {
+public function updateImage(http:Request updateImageReq, string orgName, string imageName, gen:ImageUpdateRequest updateImageBody) returns http:Response {
     if (updateImageReq.hasHeader(constants:AUTHENTICATED_USER)) {
```

Comment:
Need to check if the user has permission (in the org the image belongs to) to update the image.

Reply:
It checked within the query "UPDATE_IMAGE_DESCRIPTION_N_SUMMARY_QUERY"